Wrappers/Python/test/test_algorithms.py

```diff
@@ -1131,7 +1147,210 @@ def test_SIRT_with_TV_warm_start(self):
             sirt.x.as_array(), ig.allocate(0.25).as_array(), 3)
 
 
-class TestSPDHG(unittest.TestCase):
+class TestSPDHG(CCPiTestClass):
+    def setUp(self):
+        data = dataexample.SIMPLE_PHANTOM_2D.get(size=(20, 20))
+        self.subsets = 10
+        
+        data = dataexample.SIMPLE_PHANTOM_2D.get(size=(16, 16))
+
+        ig = data.geometry
+        ig.voxel_size_x = 0.1
+        ig.voxel_size_y = 0.1
+
+        detectors = ig.shape[0]
+        angles = np.linspace(0, np.pi, 90)
+        ag = AcquisitionGeometry.create_Parallel2D().set_angles(
+            angles, angle_unit='radian').set_panel(detectors, 0.1)
+        # Select device
+        dev = 'cpu'
```

Comment:
why is this cpu?

Reply:
I think it was slightly quicker when I was running the tests - will double check